api/everycache_api/auth/token_cleanup.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from apscheduler.triggers.interval import IntervalTrigger
+from flask import current_app
+from pytz import utc
+
+from everycache_api.extensions import apscheduler, db
+from everycache_api.models import Token
+
+
+def cleanup_expired_tokens():
+    with apscheduler.app.app_context():
+        tokens = Token.query.filter(Token.expires < datetime.utcnow()).all()
+
+        if tokens:
+            current_app.logger.info(
+                f"Found {len(tokens)} expired tokens in database. Cleaning up..."
+            )
+
+            for token in tokens:
+                db.session.delete(token)
```

Comment:
wouldn't `tokens.delete()` work?

Reply:
No, but I changed it to be more efficient anyway, see 495ec88